src/prefect/orion/models/logs.py

```diff
@@ -37,13 +42,12 @@ async def create_logs(
         None
     """
     log_insert = await db.insert(db.Log)
-    for batch in batched_iterable(logs, LOG_BATCH_SIZE):
-        await session.execute(log_insert.values([log.dict() for log in batch]))
```

Comment:
I wanted to preserve the convention that all `models` methods take a `session` argument, so I've moved the batching out in the API.